gamemodes/terrortown/gamemode/shared/sh_marker_vision_object.lua

```diff
@@ -0,0 +1,244 @@
+---
+-- @author Mineotopia
+-- @class MARKER_VISION_OBJECT
```

Comment:
I see what you are saying. The data object is named the same as the targetID one therefore the name.

I don't have a good name for the synced one (the object) but it simplifies it a lot because I can have functions on it and don't need any manual table manipulations.

I'm not a fan of merging these two as one is purely for rendering and the other is for syncing and managing.

The data object also is not per entity but one that gets changed its content per loop iteration

Reply:
I only said you should clarify this in a comment, its fine that there are 2 they are just very confusing